cogs/displayinfo.py

```diff
@@ -34,20 +34,38 @@ class DisplayInfo(commands.Cog):
     def __init__(self,bot):
         self.bot = bot
         self.current_collection = db["UserTracking"] # TODO: Move this secrets file
-        self.timers = self.current_collection.find_one() 
+        self.timers = self.current_collection.find_one() # Gets document from collection.
         self.current_guild_name = ""
         self.current_date = str(date.today())
         self.members_online = set()
         self.timed_task = None
     
+    '''
+    Sample dictionary layout: {'_id': ObjectId('64d6c8e896c982ece6550d23'), '326586435958341636': {'2023-08-14': { 'Activity 1': {'totalTime': [], 'startTime': []}, 'Activity 2': {'totalTime': [], 'startTime': []}}}, 
+    '2023-08-15': {'Activity 1': {'totalTime': [], 'startTime': []}
+    '''
+    
     @commands.Cog.listener()
     async def on_ready(self):
         for guild in self.bot.guilds:
             for member in guild.members:
-                if member.status == discord.Status.online:
+                if member.status == discord.Status.online and member.activity:
                     self.members_online.add(member)
-
-        if len(self.members_online) != 0:
+                    after_string_id = str(member.id)
```

Comment:
Might be good to have code reusability here and just call `update_now()`?

Reply:
update_now requires them to have a startTime which these members do not have yet

Reply:
I like `update_now()` being a function that can be reusable and having no issues. We should update that function to make the checks for if startTime is not present and do the logic there. A reason for that is that `update_now()` can also be called in the future at 12 oclock to immediately update/start a new timer then we can delete the previous days' startTime to save database space.

Reply:
Refer to comment on line 121